pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "astoria"
 version = "0.1.0"
-description = "Robot Management Daemon"
-authors = ["Dan Trickey <dan@trickey.io>"]
+description = "Robot Management System for Student Robotics 2022 kit"
```

Comment:
do we want to tie it so closely to the 2022 kit? (both from the perspective of "who knows what the future will bring" and also "probably nobody will remember to update it for 2023")

maybe "...for post-2020 Student Robotics kit", though, oof, kind of draws attention to Events

or just "...for Student Robotics kit"

Reply:
Agree. Have changed to `for Student Robotics Kit`

Reply:
We should change the README too